linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpIdentifierConfig.scala

```diff
@@ -1,9 +1,10 @@
-package io.l5d
+package io.buoyant.linkerd.protocol
```

Comment:
In #388 I'm moving this file to:

io/buoyant/linkerd/protocol/http/DefaultIdentifierConfigTest.scala

Instead of:

io/buoyant/linkerd/protocol/DefaultHttpIdentifierConfigTest.scala

I think I have a slight preference for the former, given that's how we handle other http-specific configuration, but could switch to the latter if you prefer.